### ISSUE TITLE
Add yarn & pnpm support

### ISSUE THREAD

**Issue by nareshbhatia** [code-shaper/code-shaper#2]
Test with yarn & pnpm and determine what, if anything, is required to support them.

**Comment by dagilleland** [comment]
Yes, this would be wonderful! For example, with pnpm, there is an error that states it doesn't/can't use the `workspaces` field in the `package.json` file, instead encouraging us to use a `pnpm-workspace.yaml`. Also, Windows uses (like myself) can't run `nvm use` and have to go to other node version managers (like `nvs`).

**Comment by nareshbhatia** [comment]
Yes, it would indeed be good to have yarn & pnpm under our belt. We initially went with npm because yarn was causing issues with turborepo. I am sure this has improved over time. In any case, we can probably at least document how to alter workflows based on these tools and also other node version managers. PRs welcome!
